src/platforms/apple/common/enriching-events/attachments/index.mdx

```diff
@@ -33,6 +31,12 @@ In addition, you can set these parameters:
 
 Attachments live on the <PlatformLink to="/enriching-events/scopes/">Scope</PlatformLink>. You can either add an attachment on the global scope to be sent with every event or add it on the <PlatformLink to="/enriching-events/scopes/#local-scopes">local Scope</PlatformLink> to just send the attachment with one specific event.
 
+<Note>
+
+Adding attachments to crashes is currently not supported.
```

Comment:
```suggestion
To add attachments to crashes, use <PlatformLink to="/configuration/options/#initial-scope">`initialScope`</PlatformLink>, and remove them after initialization of the SDK.
```

Reply:
do we want to suggest this here? @philipphofmann

Reply:
I'm not familiar with this way of adding attachments to crashes, but this sentence probably doesn't give enough context to make it possible for most users to be successful.

If we want to include this, we should definitely elaborate. I'd add a separate subheader for it.

Reply:
The problem is that you would add this whenever the app starts / users initialize the SDK. `SentrySDK.crashedLastRun` doesn't work for `initialScope` because the SDK must be initialized. I think this could confuse users more than it helps. Adding hints to the Cocoa SDK is the way to go IMO, and then we can update the docs.

Reply:
Yeah that's what I expected - thank you both